python/ray/data/tests/test_dataset_parquet.py

```diff
@@ -152,11 +153,11 @@ def test_parquet_read_basic(ray_start_regular_shared, fs, data_path):
         repr(ds) == "Datastream(num_blocks=2, num_rows=6, "
         "schema={one: int64, two: string})"
     ), ds
-    check_num_computed(ds, 1, 1)
+    check_num_computed(ds, 0, 0)
 
     # Forces a data read.
     values = [[s["one"], s["two"]] for s in ds.take_all()]
-    check_num_computed(ds, 2, 2)
+    check_num_computed(ds, 2, 0)
```

Comment:
why we have difference between bulk and streaming?

Reply:
the comment here explains: https://github.com/ray-project/ray/blob/3f9b2382d158640ecb739e9bafbe01b1d6a30369/python/ray/data/tests/test_dataset_parquet.py#L37-L40